<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BioStat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="version">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Constantia&quot;;</string>
    </property>
    <property name="text">
     <string>version 1.0.0</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textSearch">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;verdana&quot;;
/*background-color: rgb(249, 222, 149);*/
background-color: rgb(255, 255, 255);
border-radius: 2px;
border: 2px solid #9a9;
box-shadow: 4px 4px 4px rgba(8, 8, 8, 0.5);
selection-color: rgb(170, 255, 127);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'verdana'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushOkSearch">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>60</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;verdana&quot;;
/*background-color: rgb(249, 222, 149);*/
background-color: rgb(255, 255, 255);
border-radius: 2px;
border: 2px solid #9a9;
box-shadow: 4px 4px 4px rgba(8, 8, 8, 0.5);
selection-color: rgb(170, 255, 127);</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBackFromStart">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>410</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;verdana&quot;;
/*background-color: rgb(249, 222, 149);*/
background-color: rgb(255, 255, 255);
border-radius: 2px;
border: 2px solid #9a9;
box-shadow: 4px 4px 4px rgba(8, 8, 8, 0.5);
selection-color: rgb(170, 255, 127);</string>
    </property>
    <property name="text">
     <string>НАЗАД</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
